Gemfile

```diff
@@ -14,15 +14,16 @@ gem 'omniauth-rails_csrf_protection', '~> 1.0.1'
 gem 'rails', '~> 7.1.0'
 gem 'sassc-rails'
 gem 'webmock', '~> 3.23.0'
-gem 'webrick', '~> 1.8.1'
+gem 'webrick', '~> 1.8.2'
 gem 'aws-sdk', '~> 3'
 gem 'rexml', '~> 3.3.7'
+gem 'tzinfo-data'
 
 group :test do
   gem 'rack_session_access', '~> 0.2.0'
 end
 
-gem "listen", "~> 3.9", :group => :development
+gem "listen", "~> 3.9"
```

Comment:
Why did we need to do this? Looks like the listen gem is only used in the development environment.

Reply:
when we specify it with `:group => :development` in the Gemfile, our Docker build fails with the error:

"listen is not part of the bundle. Add it to your Gemfile."

Reply:
hmm, I think this is the wrong solution. I added a [comment on the Jira ticket about this](https://gds-cyber-and-engineering.atlassian.net/browse/ENG-487?focusedCommentId=12538)

we just need to specific the rails env in the final `CMD` line in the docker file

a nice way of doing that would be to add an `ENV` statement which defaults `RAILS_ENV` to `production` but allows it to be overridden if needed

Reply:
Hauwa has now put this back in so that the image will not include the listen gem in production. We can test the image locally by setting the `RAILS_ENV` env variable to `production`, and leave the `COST_CENTRE_S3_BUCKET_NAME` variable unset, which will make the app load the dummy cost centre data from the `cost_centre_fixture.csv` file instead of going to S3.